src/emulator/functionsEmulator.ts

```diff
@@ -647,6 +675,44 @@ export class FunctionsEmulator implements EmulatorInstance {
     }
   }
 
+  performPostLoadOperations(): Promise<void> {
+    if (
+      this.blockingFunctionsConfig.triggers?.beforeCreate?.functionUri === "" &&
+      this.blockingFunctionsConfig.triggers.beforeSignIn?.functionUri === ""
+    ) {
+      return Promise.resolve();
+    }
+
+    const authEmu = EmulatorRegistry.get(Emulators.AUTH);
+    if (!authEmu) {
+      return Promise.resolve();
+    }
+
+    const path = `/identitytoolkit.googleapis.com/v2/projects/${this.getProjectId()}/config?updateMask=blockingFunctions`;
+
+    return api
+      .request("PATCH", path, {
+        origin: `http://${EmulatorRegistry.getInfoHostString(authEmu.getInfo())}`,
+        headers: {
+          Authorization: "Bearer owner",
+        },
+        data: {
+          blockingFunctions: this.blockingFunctionsConfig,
+        },
+        json: true,
+      })
+      .then(() => {
+        return Promise.resolve();
+      })
```

Comment:
This `then` doesn't seem useful. Is it serving some purpose I'm not aware of?

Reply:
I was following convention from the other API requests in `functionsEmulator`, I'll remove it and make the whole function async